enterprise/server/api/api_server.go

```diff
@@ -377,6 +377,21 @@ func fillActionFromBuildEvent(action *apipb.Action, event *build_event_stream.Bu
 			action.Id.TargetId = encodeID(event.GetId().GetTestResult().GetLabel())
 			action.Id.ConfigurationId = event.GetId().GetTestResult().GetConfiguration().Id
 			action.Id.ActionId = encodeID(fmt.Sprintf("test-S_%d-R_%d-A_%d", testResultID.Shard, testResultID.Run, testResultID.Attempt))
+			return action
+		}
+	}
+	return nil
+}
+
+func fillActionOutputFilesFromBuildEvent(action *apipb.Action, event *build_event_stream.BuildEvent) *apipb.Action {
```

Comment:
typically the "output" param comes last in functions where you take both inputs and outputs as pointers